source/manual/schemas.html.md

```diff
@@ -1,41 +1,94 @@
+
+[Google also makes FAQPage content available as answers in Google assistant](https://developers.google.com/assistant/content/faq)
+
+We have a few different implementations of this right now, because it's very new and we wanted to be able to explore its use in different document types.  They are:
+
+- [The schema used on answers in GOV.UK publishing components](https://github.com/alphagov/govuk_publishing_components/blob/master/lib/govuk_publishing_components/presenters/machine_readable/faq_page_schema.rb)
+- [The schema used on most guides in Government Frontend](https://github.com/alphagov/government-frontend/blob/master/app/presenters/machine_readable/guide_faq_page_schema_presenter.rb)
+- [The schema used on "how to vote" also in Government Frontend](https://github.com/alphagov/government-frontend/blob/master/app/presenters/machine_readable/yaml_faq_page_schema_presenter.rb)
+- [The schema used for transactions in Frontend](https://github.com/alphagov/frontend/blob/master/app/presenters/machine_readable/transaction_faq_page_schema.rb)
+
+We may consolidate these at some point!
+
+### GovernmentOrganization
+
+We use [the GovernmentOrganization schema](https://schema.org/GovernmentOrganization) on [organisation pages](https://www.gov.uk/government/organisations/government-digital-service).
+
+GOV.UK is the source of info on UK government organisations (we have an API which is used to power the register).
+
+### HowTo
+
+[The HowTo schema](https://schema.org/HowTo) is used on step by step pages. We use this particularly because it can result in a rich result on mobile [displaying each step with a thumbnail image](https://search.google.com/test/rich-results?utm_campaign=devsite&utm_medium=jsonld&utm_source=how-to&id=5DXkD7BHHFDFj-QuPvlQvQ&view=search-preview).
+
+Pages using this schema appear in [the How-to "enhancements" section within the Google search console](https://search.google.com/search-console/how-to?resource_id=https%3A%2F%2Fwww.gov.uk%2F) which allows us to track whether errors have cropped up in the schema on specific pages.  This is useful because it means we don't have to manually check all the pages when we tweak things.
+
+[Google's requirements for pages that implement the HowTo schema](https://developers.google.com/search/docs/data-types/how-to).
+
+You should always use the [rich results testing tool](https://search.google.com/test/rich-results) to check changes to this schema.
+
+[The Howto schema's implemented in Collections](https://github.com/alphagov/collections/blob/master/app/models/schemas/how_to.rb)
+
+### NewsArticle
+
+[The News Article schema](https://schema.org/NewsArticle) is implemented (surprise surprise) on [news articles](https://github.com/alphagov/government-frontend/blob/11dfc12b47ede18897be28350db31ca23743c46c/app/views/content_items/news_article.html.erb#L3).
```

Comment:
This should be "The NewsArticle schema" for consistency

Reply:
👍